client/src/components/MultiJurisdictionAudit/index.test.tsx

```diff
@@ -281,6 +281,53 @@ describe('JA setup', () => {
     })
   })
 
+  it('deletes ballot manifest', async () => {
+    const expectedCalls = [
+      jaApiCalls.getUser,
+      jaApiCalls.getSettings(auditSettings.batchComparisonAll),
+      jaApiCalls.getRounds,
+      jaApiCalls.getBallotManifestFile(manifestMocks.empty),
+      jaApiCalls.getBatchTalliesFile(talliesMocks.empty),
+      jaApiCalls.getCVRSfile(cvrsMocks.empty),
+      jaApiCalls.putManifest,
+      jaApiCalls.getBallotManifestFile(manifestMocks.processed),
+      jaApiCalls.deleteManifest,
+      jaApiCalls.getBallotManifestFile(manifestMocks.empty),
+    ]
+    await withMockFetch(expectedCalls, async () => {
+      renderView()
+      await screen.findByText('Audit Source Data')
+      const [manifestInput, talliesInput] = screen.getAllByLabelText(
```

Comment:
technically, in the ballot comparison case, the second input is for cvrs, not tallies. but not a big deal